### ISSUE TITLE
helm template command failing

### ISSUE THREAD

**Issue by nickapos** [nautobot/helm-charts#249]
Hey everyone, I have a question about an issue I am having with nautobot helm chart v1.3.13 and v1.13.14
When i am running it with helm template command i am getting the following error:

```
nikolaos.a@MBP-176136 [15:44:52] [~/testing/Projects/nautobot-helm-config] [master *]
-> % helm template nautobot nautobot/nautobot -n nautobot --values helm-config-prd.yml --debug
install.go:194: [debug] Original chart version: ""
install.go:211: [debug] CHART PATH: /Users/nikolaos.a/Library/Caches/helm/repository/nautobot-1.3.14.tgz


Error: execution error at (nautobot/templates/secret.yaml:17:27): Existing secret postgres-secret not found!
helm.go:84: [debug] execution error at (nautobot/templates/secret.yaml:17:27): Existing secret postgres-secret not found!
nikolaos.a@MBP-176136 [15:58:16] [~/testing/Projects/nautobot-helm-config] [master *]
```

This happens both in the cli as well via our deployment tool. Commands helm install and helm upgrade work fine and the secret that supposedly does not exist, does in fact exist. I am a bit confused why this is happening. Any ideas?

**Comment by nniehoff** [comment]
This was fixed in #254 and will be released shortly in version 2.0.1

**Comment by X4mp** [comment]
It seems like this is actually expected behavior since helm template can't use the lookup to access external cluster resources like secrets when using `helm template`. 
If you are aware of a way to make this work locally and with ArgoCD I'd appreciate it a lot. Installing the chart with `helm install` works just fine.
According to [this blog](https://itnext.io/manage-auto-generated-secrets-in-your-helm-charts-5aee48ba6918) (see section *More on the lookup Function*) `helm template` always returns an empty response.
